### PR TITLE
Translate "Intro to css"

### PR DESCRIPTION
**Pull request by emredevsalot**

### CODE REVIEW

File: temel_bilgiler/html_css/css-e-giris.md [hunk 1]
```diff
@@ -0,0 +1,414 @@
+### Giriş
+
+Önceki derste, bir internet sayfasının nasıl yapılandırıldığını belirleyen HTML'in nasıl yazıldığını öğrendiniz. Bir sonraki adım, bu yapının biraz _stil_ ile güzel görünmesini sağlamaktır. CSS de tam bu noktada devreye girer. Sonraki birkaç derste, bazı temel CSS kavramları olduğuna inandığımız ve herkesin en başından bilmesi gereken şeylere(ister yeni başlıyor olun, ister sadece bilgilerinizi tazelemek isteyin) odaklanacağız.
+
+### Derse Genel Bakış
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
### Derse genel bakış
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+
+Bu bölüm, bu derste öğreneceğiniz konulara genel bir bakış içerir.
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
Bu bölüm, bu derste öğreneceğiniz konuların genel bir özetini içerir.
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+
+- CSS ile HTML'e stiller ekleyin.
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
- CSS ile HTML'e stiller eklemek.
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+- Class ve ID niteliklerinin nasıl kullanılacağını anlayın.
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
- Class ve ID niteliklerinin nasıl kullanılacağını anlamak.
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+- Doğru seçicileri kullanarak belirli öğelere stiller ekleyin.
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
- Doğru seçicileri kullanarak belirli öğelere stiller eklemek.
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 6]
```diff
@@ -0,0 +1,414 @@
...
+
+Daha interaktif bir açıklama ve örnek için aşağıdaki Scrim'i deneyin (bunlar hakkında ne düşündüğünüzü bize bildirin):
+
+<iframe src="https://scrimba.com/scrim/co12d4cf99cf2776f19e84a9d?embed=odin,mini-header,no-sidebar,no-next-up" sandbox="allow-scripts allow-same-origin allow-popups" width="100%" height="400"></iframe>
+
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
```
This link is updated and moved to the end of this file.

File: temel_bilgiler/html_css/css-e-giris.md [hunk 7]
```diff
@@ -0,0 +1,414 @@
...
+- <a class="knowledge-check-link" href="#chaining-selectors">ID'si `title` ve sınıfı `primary` olan bir öğe verildiğinde, tek bir CSS kuralı için her iki niteliği nasıl kullanırsınız?</a>
+- <a class="knowledge-check-link" href="#descendant-combinator-description">Soydan gelen birleştirici ne işe yarar?</a>
+- <a class="knowledge-check-link" href="#adding-css-to-html">HTML'e CSS eklemek için kullanılan üç yöntemin isimleri nelerdir?</a>
+- <a class="knowledge-check-link" href="#external-css">HTML'e CSS eklemenin üç yöntemi arasındaki temel farklar nelerdir?</a>
+
+### Ek Kaynaklar
+
+Bu alanda içerikle alakalı faydalı linkler bulunmaktadır. Zorunlu değildir, ek olarak düşünülmelidir.
+
+- [Mozilla CSS değerleri ve birimleri](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units) mutlak veya göreli olarak mümkün olan çeşitli değer türlerini öğrenmek için kullanılabilir.
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
- [Mozilla CSS values and units](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units) adlı ingilizce makale mutlak veya göreli olarak mümkün olan çeşitli değer türlerini öğrenmek için kullanılabilir.
- [Bu ingilizce interaktif Scrim](https://scrimba.com/scrim/co12d4cf99cf2776f19e84a9d) dersteki materyallerin çoğunu interaktif bir biçimde ele almaktadır.
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 8]
```diff
@@ -0,0 +1,414 @@
...
+
+- CSS ile HTML'e stiller ekleyin.
+- Class ve ID niteliklerinin nasıl kullanılacağını anlayın.
+- Doğru seçicileri kullanarak belirli öğelere stiller ekleyin.
+
+Daha interaktif bir açıklama ve örnek için aşağıdaki Scrim'i deneyin (bunlar hakkında ne düşündüğünüzü bize bildirin):
+
+<iframe src="https://scrimba.com/scrim/co12d4cf99cf2776f19e84a9d?embed=odin,mini-header,no-sidebar,no-next-up" sandbox="allow-scripts allow-same-origin allow-popups" width="100%" height="400"></iframe>
+
+### Temel Sözdizimi
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
### Temel sözdizimi
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 9]
```diff
@@ -0,0 +1,414 @@
...
+ 
+`<div>`, temel HTML öğelerinden biridir. En basit haliyle boş bir konteynerdir. Genellikle projelerinizdeki içerik için `<h1>` veya `<p>` gibi diğer etiketleri kullanmak en iyisidir. Ancak CSS hakkında daha fazla bilgi edindikçe, birçok durumda ihtiyacınız olan şeyin, yalnızca diğer öğeleri barındırmanızı sağlayan bir konteyner olduğunu göreceksiniz. Alıştırmalarımızın çoğunda basitlik için `<div>`ler kullanılmaktadır. Daha sonraki derslerde, çeşitli HTML öğelerinin ne zaman kullanılması gerektiği konusunda çok daha derinlemesine bilgi verilecektir.
+
+</div>
+
+### Seçiciler(Selectors)
+
+Seçiciler, CSS kurallarının uygulanması gereken HTML öğelerini belirtir. Her kural için hedefleri tanımlarlar. Aşağıdaki bölümlerde tüm seçicilere yer verilmemiştir. Ancak en yaygın ve başlangıçta rahatça kullanmanız gerekenler bunlardır.
+
+#### Evrensel Seçici
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
#### Evrensel seçici
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 10]
```diff
@@ -0,0 +1,414 @@
...
+
+Evrensel seçici her türden öğeyi seçer, bu nedenle "evrensel"dir ve sözdizimi bir yıldız işaretidir. Aşağıdaki örnekte, her öğeye `color: purple;` stili uygulanacaktır.
+
+~~~css
+* {
+  color: purple;
+}
+~~~
+
+#### Tip Seçiciler
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
#### Tip seçiciler
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 11]
```diff
@@ -0,0 +1,414 @@
...
+/* styles.css */
+
+div {
+  color: white;
+}
+~~~
+
+Burada, üç `<div>` öğesi de seçilirken, `<p>` öğesi seçilmez.
+
+#### Sınıf Seçiciler
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
#### Class(sınıf) seçiciler
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 12]
```diff
@@ -0,0 +1,414 @@
...
+
+Sınıf seçiciler, aynı sınıfa(HTML öğelerine verilen niteliklerden biri) sahip tüm HTML öğelerini seçerler. Bir HTML öğesine nasıl sınıf ekleyeceğiniz ve bu sınıfı CSS'te nasıl seçeceğiniz aşağıda açıklanmıştır:
```
Comment (huseyingulec) [inline temel_bilgiler/html_css/css-e-giris.md:70]:
```suggestion
Class(sınıf) seçiciler, aynı sınıfa(HTML öğelerine verilen niteliklerden biri) sahip tüm HTML öğelerini seçerler. Bir HTML öğesine nasıl class ekleyeceğiniz ve bu class'ı CSS'te nasıl seçeceğiniz aşağıda açıklanmıştır:
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 13]
```diff
@@ -0,0 +1,414 @@
...
+
+~~~css
+/* styles.css */
+
+.alert-text {
+  color: red;
+}
+~~~
+
+Sınıf seçicilerin sözdizimine dikkat edin. Bir nokta ve hemen ardından hedeflenen sınıfın büyük/küçük harfe duyarlı değeri olarak kullanılır. Sınıfların belirli bir öğeye özel olması gerekmez. Bu nedenle aynı sınıfı istediğiniz sayıda öğeye ekleyebilirsiniz.
```
Comment (huseyingulec) [inline temel_bilgiler/html_css/css-e-giris.md:86]:
```suggestion
Class seçicilerin sözdizimine dikkat edin. Bir nokta ve hemen ardından hedeflenen sınıfın büyük/küçük harfe duyarlı değeri olarak kullanılır. Class'ların belirli bir öğeye özel olması gerekmez. Bu nedenle aynı class'ı istediğiniz sayıda öğeye ekleyebilirsiniz.
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 14]
```diff
@@ -0,0 +1,414 @@
...
+
+Sınıflarla yapabileceğiniz bir başka şey ise tek bir öğeye birden fazla sınıfı aralarında boşluk bırakarak liste halinde eklemektir. Örneğin `class="alert-text severe-alert"` şeklinde kullanabilirsiniz. Sınıfları ayırmak amacıyla boşluk kullanıldığı için, birden fazla kelimeyle tanımlanabilecek sınıfların isimlerinde asla boşluk kullanmamalı, bunun yerine kısa çizgi kullanılmalıdır.
```
Comment (huseyingulec) [inline temel_bilgiler/html_css/css-e-giris.md:88]:
```suggestion
Class'larla yapabileceğiniz bir başka şey ise tek bir öğeye birden fazla class'ı aralarında boşluk bırakarak liste halinde eklemektir. Örneğin `class="alert-text severe-alert"` şeklinde kullanabilirsiniz. Class'ları ayırmak amacıyla boşluk kullanıldığı için, birden fazla kelimeyle tanımlanabilecek class'ların isimlerinde asla boşluk kullanmamalı, bunun yerine kısa çizgi kullanılmalıdır.
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 15]
```diff
@@ -0,0 +1,414 @@
...
+
+#### ID Seçiciler
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
#### ID seçiciler
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 16]
```diff
@@ -0,0 +1,414 @@
...
+
+ID seçiciler sınıf seçicilere benzer. HTML öğelerinin başka bir niteliği olan ID'ye sahip bir öğeyi seçerler. Sınıflar ve ID'ler arasındaki en büyük fark, bir öğenin yalnızca **bir** ID'ye sahip olabilmesidir. Bir sayfada tekrarlanamaz ve adında herhangi bir boşluk içermemelidir:
```
Comment (huseyingulec) [inline temel_bilgiler/html_css/css-e-giris.md:92]:
```suggestion
ID seçiciler class seçicilere benzer. HTML öğelerinin başka bir niteliği olan ID'ye sahip bir öğeyi seçerler. Class'lar ve ID'ler arasındaki en büyük fark, bir öğenin yalnızca **bir** ID'ye sahip olabilmesidir. Bir sayfada tekrarlanamaz ve adında herhangi bir boşluk içermemelidir:
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 17]
```diff
@@ -0,0 +1,414 @@
...
+
+~~~css
+/* styles.css */
+
+#title {
+  background-color: red;
+}
+~~~
+
+ID'ler için nokta yerine hashtag kullanılır. Hemen ardından ID'nin büyük/küçük harfe duyarlı değeri gelir. Sıkça yapılan bir hata, insanların ihtiyaç duymadığı ve sınıfların yeterli olacağı durumlarda ID özelliğini aşırı kullanmasıdır. ID kullanmanın anlamlı veya gerekli olduğu durumlar olsa da(örneğin, özgüllüğün avantajından yararlanmak veya bağlantıların geçerli sayfadaki bir bölüme yönlendirmesi) onları **idareli** kullanmalısınız.
```
Comment (huseyingulec) [inline temel_bilgiler/html_css/css-e-giris.md:108]:
```suggestion
ID'ler için nokta yerine hashtag kullanılır. Hemen ardından ID'nin büyük/küçük harfe duyarlı değeri gelir. Sıkça yapılan bir hata, insanların ihtiyaç duymadığı ve class'ların yeterli olacağı durumlarda ID özelliğini aşırı kullanmasıdır. ID kullanmanın anlamlı veya gerekli olduğu durumlar olsa da(örneğin, özgüllüğün avantajından yararlanmak veya bağlantıların geçerli sayfadaki bir bölüme yönlendirmesi) onları **idareli** kullanmalısınız.
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 18]
```diff
@@ -0,0 +1,414 @@
...
+
+#### Gruplandırma seçici
+
+Peki ya stil tanımlamalarından bazılarını paylaşan iki grup öğemiz varsa ne yapmalıyız?
+
+~~~css
+.read {
+  color: white;
+  background-color: black;
+  /* bu sınıfa özel tanımlamalar */
```
Comment (huseyingulec) [inline temel_bilgiler/html_css/css-e-giris.md:118]:
```suggestion
  /* bu class'a özel tanımlamalar */
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 19]
```diff
@@ -0,0 +1,414 @@
...
+}
+
+.unread {
+  color: white;
+  background-color: black;
+  /* bu sınıfa özel tanımlamalar */
```
Comment (huseyingulec) [inline temel_bilgiler/html_css/css-e-giris.md:124]:
```suggestion
  /* bu class'a özel tanımlamalar */
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 20]
```diff
@@ -0,0 +1,414 @@
...
+
+~~~css
+.read,
+.unread {
+  color: white;
+  background-color: black;
+}
+
+.read {
+  /* bu sınıfa özel tanımlamalar */
```
Comment (huseyingulec) [inline temel_bilgiler/html_css/css-e-giris.md:138]:
```suggestion
  /* bu class'a özel tanımlamalar */
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 21]
```diff
@@ -0,0 +1,414 @@
...
+}
+
+.unread {
+  /* bu sınıfa özel tanımlamalar */
```
Comment (huseyingulec) [inline temel_bilgiler/html_css/css-e-giris.md:142]:
```suggestion
  /* bu class'a özel tanımlamalar */
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 22]
```diff
@@ -0,0 +1,414 @@
...
+}
+~~~
+
+Yukarıdaki örneklerin her ikisi de (gruplandırmalı ve gruplandırmasız) aynı sonuca sahip olacaktır. Ancak ikinci örnek, tanımlamaların tekrarını azaltır ve her iki sınıf için `color` ve `background-color`ın aynı anda düzenlenmesini kolaylaştırır.
```
Comment (huseyingulec) [inline temel_bilgiler/html_css/css-e-giris.md:146]:
```suggestion
Yukarıdaki örneklerin her ikisi de (gruplandırmalı ve gruplandırmasız) aynı sonuca sahip olacaktır. Ancak ikinci örnek, tanımlamaların tekrarını azaltır ve her iki class için `color` ve `background-color`ın aynı anda düzenlenmesini kolaylaştırır.
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 23]
```diff
@@ -0,0 +1,414 @@
...
+}
+
+.subsection#preview {
+  color: blue;
+}
+~~~
+
+Genel olarak, bir öğe aynı anda iki farklı tipte olamayacağı için birden fazla tip seçiciyi zincirleyemezsiniz. Örneğin, `div` ve `p` gibi iki tip seçiciyi zincirlemek, bize `divp` seçicisini verir. Seçici, var olmayan bir `<divp>` öğesi bulmaya çalışacağı için işe yaramaz.
+
+#### Soydan Gelen Birleştirici
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
#### Soydan gelen birleştirici
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 24]
```diff
@@ -0,0 +1,414 @@
...
+
+~~~css
+/* styles.css */
+
+.ancestor .contents {
+  /* some declarations */
+}
+~~~
+
+Yukarıdaki örnekte, `contents` sınıfına sahip ilk iki öğe (B ve C) seçilecektir. Ancak son öğe (D) seçilmeyecektir. Tahmininiz doğru muydu?
```
Comment (huseyingulec) [inline temel_bilgiler/html_css/css-e-giris.md:223]:
```suggestion
Yukarıdaki örnekte, `contents` class'ına sahip ilk iki öğe (B ve C) seçilecektir. Ancak son öğe (D) seçilmeyecektir. Tahmininiz doğru muydu?
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 25]
```diff
@@ -0,0 +1,414 @@
...
+
+Bir kurala kaç tane birleştirici ekleyebileceğiniz konusunda net bir sınır yoktur. Bu nedenle `.one .two .three .four` tamamen geçerli olacaktır. Bu seçim, `four` sınıfına sahip öğeyi ancak `three` sınıfına sahip bir atası varsa, o atanın `two`, onun atasının da `one` sınıfına sahip öğesi varsa seçecektir. Genellikle bu seviyede iç içe geçmeye ihtiyaç duyan öğeleri seçmeye çalışmaktan kaçınmak isteriz. Çünkü oldukça kafa karıştırıcı ve uzun olabilirler. Özgüllük konusunda sorunlara da neden olabilirler.
```
Comment (huseyingulec) [inline temel_bilgiler/html_css/css-e-giris.md:225]:
```suggestion
Bir kurala kaç tane birleştirici ekleyebileceğiniz konusunda net bir sınır yoktur. Bu nedenle `.one .two .three .four` tamamen geçerli olacaktır. Bu seçim, `four` class'ına sahip öğeyi ancak `three` class'ına sahip bir atası varsa, o atanın `two`, onun atasının da `one` class'ına sahip öğesi varsa seçecektir. Genellikle bu seviyede iç içe geçmeye ihtiyaç duyan öğeleri seçmeye çalışmaktan kaçınmak isteriz. Çünkü oldukça kafa karıştırıcı ve uzun olabilirler. Özgüllük konusunda sorunlara da neden olabilirler.
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 26]
```diff
@@ -0,0 +1,414 @@
...
+  font-size: 32px;
+}
+.last_declared {
+  font-size: 14px;
+  font-weight: 800;
+}
+~~~
+
+Yukarıdaki örnekte, `.last_declared`(son tanımlanan) seçicisi `.first_declared`(ilk tanımlanan) seçicisini geçersiz kılacaktır. Bu sınıfların her ikisine de sahip olan herhangi bir html öğesi, `.first_declared` yerine `.last_declared`da tanımlanan stillere sahip olacaktır.
+
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
```
This is also removed.

File: temel_bilgiler/html_css/css-e-giris.md [hunk 27]
```diff
@@ -0,0 +1,414 @@
...
+  color: rgb(100, 0, 127);
+}
+
+p {
+  /* hsl örneği: */
+  color: hsl(15, 82%, 56%);
+}
+~~~
+
+Bir alfa değeri ekleyerek bu renklerin opaklığını nasıl ayarlayabileceğinizi görmek için [CSS Yasal Renk Değerleri](https://www.w3schools.com/cssref/css_colors_legal.asp) adresine hızlıca göz atın.
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
Bir alfa değeri ekleyerek bu renklerin opaklığını nasıl ayarlayabileceğinizi görmek için [CSS Legal Color Values](https://www.w3schools.com/cssref/css_colors_legal.asp) adlı ingilizce makaleye göz atın.
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 28]
```diff
@@ -0,0 +1,414 @@
...
+  </style>
+</head>
+<body>
+  ...
+</body>
+~~~
+
+Bu yöntem, bir internet sitesinin _tek bir sayfasına_ benzersiz stiller eklemek için yararlı olabilir. Ancak harici yöntem gibi her şeyi ayrı tutmaz. Kural ve tanımlama sayısına bağlı olarak HTML dosyasının oldukça büyük olmasına neden olabilir.
+
+#### Satır İçi CSS
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
#### Satır içi CSS
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 29]
```diff
@@ -0,0 +1,414 @@
...
+
+- Tek bir öğeye _çok_ sayıda tanımlama eklemeye başladığınızda HTML dosyanız gereksiz yere şişmeye başlar. Böylece projeniz hızlıca dağınık bir hale gelebilir.
+- Aynı stili birçok öğeye eklemek isterseniz o stili kopyalayıp her bir öğeye yapıştırmanız gerekir. Bu da çok sayıda gereksiz tekrara ve daha fazla şişkinliğe neden olur.
+- Herhangi bir satır içi CSS, diğer iki yöntemi geçersiz kılar. Bu da beklenmedik sonuçlara neden olabilir. (Burada bu konuya girmeyecek olsak da aslında bundan faydanabileceğimiz durumlar da vardır).
+
+### Ödev
+
+<div class="lesson-content__panel" markdown="1">
+
+1.  [CSS alıştırmaları repository'mize](https://github.com/TheOdinProject/css-exercises) gidin, README dosyasını okuyun ve `01-css-methods` ile başlayıp `05-descendant-combinator` ile bitecek şekilde `foundations` dizinindeki alıştırmaları sırasıyla yapın.
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
1.  [Odin Project CSS alıştırmaları repository'sine](https://github.com/TheOdinProject/css-exercises) gidin, aşağıdaki alıştırmaları sırasıyla tamamlamadan önce her bir README dosyasını gözden geçirin:
    - `01-css-methods`
    - `02-class-id-selectors`
    - `03-group-selectors`
    - `04-chain-selectors`
    - `05-descendant-combinator`

   Not: Bu alıştırmaların çözümleri her alıştırmanın `solution` klasöründe bulunabilir.
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 30]
```diff
@@ -0,0 +1,414 @@
...
+
+</div>
+
+### Bilgi Ölçme
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
### Bilgi ölçme
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 31]
```diff
@@ -0,0 +1,414 @@
...
+
+Bu bölüm, dersi anlayıp anlamadığınızı kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta zorlanıyorsanız, soruya tıklayıp yönlendirdiği materyali inceleyin.
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
Bu bölüm, bu dersi kendi kendinize anlayıp anlamadığınızı kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta zorlanıyorsanız, soruya tıklayın ve bağlantılı olduğu materyali gözden geçirin.
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 32]
```diff
@@ -0,0 +1,414 @@
...
+
+- <a class="knowledge-check-link" href="#class-selectors">Sınıf ve ID seçiciler için sözdizimi nasıldır?</a>
+- <a class="knowledge-check-link" href="#the-grouping-selector">Tek bir kuralı iki farklı seçiciye nasıl uygularsınız?</a>
+- <a class="knowledge-check-link" href="#chaining-selectors">ID'si `title` ve sınıfı `primary` olan bir öğe verildiğinde, tek bir CSS kuralı için her iki niteliği nasıl kullanırsınız?</a>
+- <a class="knowledge-check-link" href="#descendant-combinator-description">Soydan gelen birleştirici ne işe yarar?</a>
+- <a class="knowledge-check-link" href="#adding-css-to-html">HTML'e CSS eklemek için kullanılan üç yöntemin isimleri nelerdir?</a>
+- <a class="knowledge-check-link" href="#external-css">HTML'e CSS eklemenin üç yöntemi arasındaki temel farklar nelerdir?</a>
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
- [Sınıf ve ID seçiciler için sözdizimi nasıldır?](#class-selectors)
- [Tek bir kuralı iki farklı seçiciye nasıl uygularsınız?](#the-grouping-selector)
- [ID'si `title` ve sınıfı `primary` olan bir öğe verildiğinde, tek bir CSS kuralı için her iki niteliği nasıl kullanırsınız?](#chaining-selectors)
- [Soydan gelen birleştirici ne işe yarar?](#descendant-combinator-description)
- [HTML'e CSS eklemek için kullanılan üç yöntemin isimleri nelerdir?](#adding-css-to-html)
- [HTML'e CSS eklemenin üç yöntemi arasındaki temel farklar nelerdir?](#external-css)
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 33]
```diff
@@ -0,0 +1,414 @@
...
+
+### Ek Kaynaklar
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
### Ek kaynaklar
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 34]
```diff
@@ -0,0 +1,414 @@
...
+
+#### Harici CSS
+
+Harici CSS karşılaşacağınız en yaygın yöntemdir. CSS için ayrı bir dosya oluşturmayı ve bunu HTML'in `<head>` etiketlerinin içine kendi kendini kapatan bir `<link>` öğesiyle bağlamayı içerir:
+
+~~~html
+<!-- index.html -->
+
+<head>
+  <link rel="stylesheet" href="styles.css" />
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
  <link rel="stylesheet" href="styles.css">
```
This is removed from original file.

File: temel_bilgiler/html_css/css-e-giris.md [hunk 35]
```diff
@@ -0,0 +1,414 @@
...
+}
+~~~
+
+Bir alfa değeri ekleyerek bu renklerin opaklığını nasıl ayarlayabileceğinizi görmek için [CSS Yasal Renk Değerleri](https://www.w3schools.com/cssref/css_colors_legal.asp) adresine hızlıca göz atın.
+
+#### Tipografi temelleri ve metin hizalama
+
+`font-family`(yazı tipi ailesi) tek bir değer olabileceği gibi, bir öğenin hangi yazı tipini kullanacağını belirleyen değerlerin bir listesi de olabilir. Her yazı tipi iki kategoriden birine girer, ya `"Times New Roman"` gibi bir "yazı tipi ailesi adı"(kelimeler arasındaki boşluklar nedeniyle tırnak işareti kullanırız) ya da `sans-serif` gibi bir "genel aile adı"dır(genel aile adlarında asla tırnak işareti kullanılmaz).
+
+Bir tarayıcı, listedeki ilk yazı tipini bulamazsa veya onu desteklemiyorsa, desteklenen ve geçerli bir yazı tipi bulana kadar listeden sonraki tiplerini deneyerek devam eder. Bu nedenle, en çok kullanılmasını istediğiniz yazı tipiyle başlayıp, yedek olarak genel bir yazı tipi ailesiyle biten bir liste oluşturmak en iyisidir, örneğin `font-family: "Times New Roman", sans-serif;`
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
Bir tarayıcı, listedeki ilk yazı tipini bulamazsa veya onu desteklemiyorsa, desteklenen ve geçerli bir yazı tipi bulana kadar listeden sonraki tiplerini deneyerek devam eder. Bu nedenle, en çok kullanılmasını istediğiniz yazı tipiyle başlayıp, yedek olarak genel bir yazı tipi ailesiyle biten bir liste oluşturmak en iyisidir, örneğin `font-family: "Times New Roman", serif;`
```
Update from original file

File: temel_bilgiler/html_css/css-e-giris.md [hunk 36]
```diff
@@ -0,0 +1,414 @@
...
+
+### Seçiciler(Selectors)
+
+Seçiciler, CSS kurallarının uygulanması gereken HTML öğelerini belirtir. Her kural için hedefleri tanımlarlar. Aşağıdaki bölümlerde tüm seçicilere yer verilmemiştir. Ancak en yaygın ve başlangıçta rahatça kullanmanız gerekenler bunlardır.
+
+#### Evrensel Seçici
+
+Evrensel seçici her türden öğeyi seçer, bu nedenle "evrensel"dir ve sözdizimi bir yıldız işaretidir. Aşağıdaki örnekte, her öğeye `color: purple;` stili uygulanacaktır.
+
+~~~css
```
Comment (huseyingulec) [inline temel_bilgiler/html_css/css-e-giris.md:39]:
```suggestion
```css
```

**Review by gulech — CHANGES_REQUESTED** [review]
Thank you Emre, good job. I only tried to check updates from original file and make some changes in this file. 

Very last thing, we have [this commit](https://github.com/TheOdinProject/curriculum/commit/f041518794c3637d825654db400ea9d149f3e4d7), somehow it kept ignoring my comments when updating backticks. Can you do that for me?

**Comment by emredevsalot** [comment]
Thanks @huseyingulec 

I tried to do the changes related to code blocks here in this commit: https://github.com/kamp-us/turkce-odin-project/pull/43/commits/b23e6b46c57eb81ad5f4851b20bbc5a2081408e1 , can you check it out as well?

**Comment by hueypov** [comment]
Thank you @emredevsalot, everything look great!. Thank you for all!!

File: temel_bilgiler/html_css/css-e-giris.md [hunk 37]
```diff
@@ -0,0 +1,396 @@
...
+
+1.  [Odin Project CSS alıştırmaları repository'sine](https://github.com/TheOdinProject/css-exercises) gidin, aşağıdaki alıştırmaları sırasıyla tamamlamadan önce her bir README dosyasını gözden geçirin:
+    - `01-css-methods`
+    - `02-class-id-selectors`
+    - `03-group-selectors`
+    - `04-chain-selectors`
+    - `05-descendant-combinator`
+
+   Not: Bu alıştırmaların çözümleri her alıştırmanın `solution` klasöründe bulunabilir.
+
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
```

File: temel_bilgiler/html_css/css-e-giris.md [hunk 38]
```diff
@@ -0,0 +1,395 @@
...
+/* styles.css */
+
+div {
+  color: white;
+}
+```
+
+Burada, üç `<div>` öğesi de seçilirken, `<p>` öğesi seçilmez.
+
+#### Class(sınıf) seçiciler
```
Comment (hueypov) [inline temel_bilgiler/html_css/css-e-giris.md]:
```suggestion
#### Sınıf(class) seçiciler
```

**Pull request merged**